app/javascript/utils/VaccineAdvancedFilters.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+
+/**
+ * Transforms the vaccine_standards configuration in /config/vaccines.yml to match structure in app/javascript/data/advancedFilterOptions.js
+ * @param {vaccine_standards} vaccine_standards - vaccine_standards object
+ */
+const getVaccineAdvancedFilters = (vaccine_standards) => {
+  return {
+    name: 'vaccination',
+    title: 'Vaccination (Combination)',
+    description: 'Monitorees with specified Vaccination criteria',
+    type: 'combination',
+    tooltip:'Returns records that contain at least one Vaccination entry that meets all user-specified criteria (e.g., searching for a specific Vaccination Product Name and Administration Date will only return records containing at least one Vaccination entry with matching values in both fields).',
+    fields: getVaccineAdvancedFilterFields(vaccine_standards),
+  }
+}
+
+const getVaccineAdvancedFilterFields = (vaccine_standards) => {
+  let vaccineGroupOptions = [];
+  let productNameOptions = [];
+  let doseNumberOptions = [];
+  let fields = [];
+
+  for (const [, value] of Object.entries(vaccine_standards)) {
+    vaccineGroupOptions.push(value["name"]);
+    productNameOptions = productNameOptions.concat(_.map(value["vaccines"], "product_name"));
+    doseNumberOptions = doseNumberOptions.concat(_.map(value["vaccines"], "num_doses"));
```

Comment:
One issue here- for the dose number options, we should instead be taking the max number from here and then using every integer from 1-MAX, since the `num_doses` property in the config is really a maximum. As it stands, if I have one vaccine in config with 3 doses, it will only give me the option of 3 or Unknown.